### PR TITLE
Fix console errors

### LINKED ISSUE #2907: Console Errors
- [x] When clicking on almost any dropdown

```javascript
content.js:4477 Uncaught TypeError: Cannot read property 'getBoundingClientRect' of null
    at content.js:4477
```

Go to any PR start commenting and refresh the page
```javascript
content.js:808 Uncaught (in promise) TypeError: elements is not iterable
    at Function.watchAndFit [as watch] (content.js:808)
    at enable (content.js:7545)
    at fitPrCommitMessageBox (content.js:7553)
    at HTMLDocument.<anonymous> (content.js:5818)
watchAndFit @ content.js:808
enable @ content.js:7545
fitPrCommitMessageBox @ content.js:7553
(anonymous) @ content.js:5818
async function (async)
(anonymous) @ content.js:5817
so @ session-resume.esm.js:144
(anonymous) @ session-resume.ts:33
```
```javascript
content.js:8660 Uncaught (in promise) TypeError: Cannot read property 'dispatchEvent' of null
    at triggerValidation (content.js:8660)
    at HTMLDocument.<anonymous> (content.js:5818)
```

The first time you go to any repo

```javascript
content.js:12768 ❌ Refined GitHub: feature `repo-age`

 TypeError: Cannot read property 'src' of undefined
    at content.js:10313
    at async getSet (content.js:2412) 
```

- [x] Press <kbd>T</kbd> on any repo issue page.

```javascript
content.js:10791 Uncaught (in promise) TypeError: Cannot read property 'rghFileFinderTerm' of null
    at setValueInField (content.js:10791)
    at preserve_file_finder_term_init (content.js:10800)
    at features_run (content.js:2798)
    at content.js:2853
    at content.js:2742
```
- [x] When editing and creating a new milestone
```js
❌ Refined GitHub → filter-comments-by-you → TypeError: Cannot read property 'cloneNode' of null
    at filter_comments_by_you_init (refined-github.js:8380)
    at features_run (refined-github.js:5917)
    at refined-github.js:5972
    at refined-github.js:5861 
Search issue: https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+Cannot%20read%20property%20'cloneNode'%20of%20null

refined-github.js:4113 ❌ Refined GitHub → hide-issue-list-autocomplete → TypeError: Cannot read property 'setAttribute' of null
    at hide_issue_list_autocomplete_init (refined-github.js:9478)
    at features_run (refined-github.js:5917)
    at refined-github.js:5972
    at refined-github.js:5861 
```
The issue is here
https://github.com/sindresorhus/refined-github/blob/d757f50ba6ae5b01638f572fcd59665706e0cdf5/source/libs/page-detect.ts#L260

### PR DESCRIPTION
**Pull request by yakov116**
Should fix #2907

I dont like it since it will make silent errors.
<!--

Thanks for contributing! 🍄

1. LINKED ISSUES:
   Does this PR close/fix an existing issue? Write something like `Closes #10`

2. TEST URLS:
   Add some test URLs

3. SCREENSHOT:
   Add a screenshot here if your PR makes visual changes

### CODE REVIEW

File: source/features/fit-textareas.tsx [hunk 1]
```diff
@@ -10,6 +10,10 @@ function inputListener(event: Event): void {
 }
 
 function watchTextarea(textarea: HTMLTextAreaElement): void {
+	if (!textarea) {
+		return;
+	}
+
```
Comment (fregante) [inline source/features/fit-textareas.tsx]: This should never be possible. This function has to be called with `textarea`. 

Only `fitPrCommitMessageBox` calls it with a possibly-non-existing element (via `select`) and even then it shouldn’t: how can the commit message field be missing from the merge box?

My guess is that onPrMergePanelOpen is firing at the wrong time. *That* needs to be fixed. 

In fact the other function has the same exact problem and the same exact solution.

Comment (yakov116) [reply source/features/fit-textareas.tsx]: >My guess is that onPrMergePanelOpen is firing at the wrong time. That needs to be fixed.

Yes that is the issue.

Any idea's i should look into?

Comment (yakov116) [reply source/features/fit-textareas.tsx]: https://github.com/sindresorhus/refined-github/blob/7f1f2c52d8deeb0fddb7a7322af7dd090ac19461/source/libs/on-pr-merge-panel-open.ts#L12

This was supposed to do that

**Comment by yakov116** [comment]
test

**Pull request closed without merging**